src/bulkDataRequestCache.ts

```diff
@@ -175,3 +187,106 @@ export class BulkUpsertRequestCache extends BulkDataRequestCache {
     await cache.write();
   }
 }
+
+export class BulkExportRequestCache extends TTLConfig<TTLConfig.Options, BulkExportCacheConfig> {
+  public static getDefaultOptions(): TTLConfig.Options {
+    return {
+      isGlobal: true,
+      isState: true,
+      filename: BulkExportRequestCache.getFileName(),
+      stateFolder: Global.SF_STATE_FOLDER,
+      ttl: Duration.days(7),
+    };
+  }
+
+  public static getFileName(): string {
+    return 'bulk-data-export-cache.json';
+  }
+
+  public static async unset(key: string): Promise<void> {
+    const cache = await BulkUpsertRequestCache.create();
+    cache.unset(key);
+    await cache.write();
+  }
+
+  /**
+   * Creates a new bulk export request cache entry for the given id.
+   */
+  public async createCacheEntryForRequest(
+    bulkRequestId: string,
+    outputInfo: {
+      filePath: string;
+      format: 'csv' | 'json';
+      columnDelimiter: ColumnDelimiterKeys;
+    },
+    username: string | undefined,
+    apiVersion: string | undefined
+  ): Promise<void> {
+    if (!username) {
+      throw messages.createError('usernameRequired');
+    }
+    this.set(bulkRequestId, {
+      jobId: bulkRequestId,
+      outputInfo,
+      username,
+      apiVersion,
+    });
+    await this.write();
+    Logger.childFromRoot('BulkExportCache').debug(`bulk cache saved for ${bulkRequestId}`);
+  }
+
+  public async resolveResumeOptionsFromCache(
+    bulkJobId: string | undefined,
+    useMostRecent: boolean,
+    apiVersion: string | undefined
+  ): Promise<ResumeBulkExportOptions> {
+    if (!useMostRecent && !bulkJobId) {
+      throw messages.createError('bulkRequestIdRequiredWhenNotUsingMostRecent');
```

Comment:
This is going to have an error name of "BulkRequestIdRequiredWhenNotUsingMostRecentError" in the output, which is a bit long.  Also, I don't see this entry in the messages file.

Reply:
oh right, I think we might be able to remote this error, the msg is defined here:
https://github.com/salesforcecli/plugin-data/blob/cd/bulk-export/messages/messages.md#bulkrequestidrequiredwhennotusingmostrecent